aws/logs_monitoring/lambda_function.py

```diff
@@ -307,6 +307,26 @@ def compileRegex(rule, pattern):
 DD_FORWARDER_VERSION = "3.15.0"
 
 
+# Used to identify and assign sources to logs
+LOG_SOURCE_SUBSTRINGS = [
```

Comment:
I don't have evidence to support this at the moment, but it feels like all of these would probably be preceded by a `/` or a `.` of some sort. Is it worth including those in the substrings in order to make similar errors less likely?

Reply:
Good point, I thought the same thing but I didn't have time to find examples of all the event types to confirm so I left it for now to prevent introducing any new bugs. But I do agree we should refactor this to be more thorough, I just made this card in triage https://datadoghq.atlassian.net/jira/software/projects/SLS/boards/205?selectedIssue=SLS-617